citi-onboarding/iJunior-onepage
# Feature/MDelivery-backend

## Back-end Modular Delivery

- Back-end Model of Modular Delivery
- Back-end Route of Modular Delivery
- Back-end Controller of Modular Delivery

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. git checkout develop
   1. git pull
   1. git checkout feature/MDelivery-backend
   1. git pull origin feature/MDelivery-backend
   1. cd ./server
   1. yarn install
   1. node index.js

1. In VScode open server

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
*
Time spent: <see your time spent on https://wakatime.com/dashboard >

Resolves:** back-end of Modular Delivery section
